domain/application/modelmigration/import.go

```diff
@@ -633,6 +646,58 @@ func (i *importOperation) importCharmActions(data description.CharmActions) (*in
 	}, nil
 }
 
+func (i *importOperation) importExposedEndpoints(ctx context.Context, app description.Application, spaces []description.Space) (map[string]application.ExposedEndpoint, error) {
+	if !app.Exposed() {
+		return nil, nil
```

Comment:
We declared that returning a nil map can have pitfalls in this PR, yet we do it here.

Reply:
yeah good call, fixed